### ISSUE TITLE
credentials.properties file is not working

### ISSUE THREAD

**Issue by binnes** [cloud-native-toolkit/iascable#188]
The environment variables defined in the credentials.properties file are not being handled properly.  Using the following workflow:

1. create a BOM (bom.yaml)
2. run iascable build -i bom.yaml
3. cd into the generated output directory
4. create a credentials.properties file in the output directory
5. run launch.sh to launch and enter the tooling container
6. cd into the bom directory and run apply.sh to deploy the BOM,  This will prompt for any missing variables

with a BOM file:

```
apiVersion: [cloudnativetoolkit.dev/v1alpha1](http://cloudnativetoolkit.dev/v1alpha1)
kind: BillOfMaterial
metadata:
  name: my-ibm-vpc-gitops
spec:
  modules:
    - name: ibm-vpc
    - name: ibm-vpc-subnets
    - name: ibm-vpc-gateways
    - name: ibm-ocp-vpc
      variables:
        - name: worker_count
          value: 1
    - name: gitops-repo
    - name: argocd-bootstrap
```

and credentials.properties file:

```
export TF_VAR_gitops_repo_username="binnes"
export TF_VAR_gitops_repo_token="XXXXXXXXX"
export TF_VAR_ibmcloud_api_key="XXXXXXXXX"
```

the environment variables are getting written to the variables.yaml files as 👍 

```
  - name: gitops_repo_username
    value: TF_VAR_gitops_repo_username=binnes
```

instead of the value just being the value of the environment variable `binnes` in the example above.

iascable verision : 2.17.0, running on an Intel Mac, MacOS : 12.5.1

**Comment by timroster** [comment]
Issue caused by parsing error in: https://github.com/cloud-native-toolkit/iascable/blob/80d8a6f4471750c4c2f9fa377a07afdbb69f01e0/scripts/apply.sh#L37

the updated `launch.sh` strips double quotes from the environment variables injected into the container causing the regex to fail.

**Comment by timroster** [comment]
fixed with #190

**Comment by triceam** [comment]
For reference, docker env files don't support quotes, but bash export statements do.  That's why we strip them in the launch script.
